2000-09-01  Jakub Jelinek  <jakub@redhat.com>

	* gcc.dg/noncompile/20000901-2.c: New test.

--- gcc/testsuite/gcc.dg/noncompile/20000901-2.c.jj	Fri Sep  1 17:11:43 2000
+++ gcc/testsuite/gcc.dg/noncompile/20000901-2.c	Fri Sep  1 17:17:44 2000
@@ -0,0 +1,18 @@
+unsigned char c;
+
+int foo()
+{
+  switch (c)
+    {
+      case 0:
+	switch (a->b)	/* { dg-error "undeclared|for each function" } */
+	  {
+	    case 0:	/* { dg-error "duplicate case" } */
+	      break;
+	  }
+}
+void bar()		/* { dg-error "parse error" } */
+{
+  n = 0;
+}
+
